docs/modules/string.md

```diff
@@ -103,11 +103,51 @@ regular expression, like this:
 
 ***
 
+## String.charAt
+
+Given a string and an index, returns the character at that index in the string.
```

Comment:
What is the return value for out-of-bounds index? Are negative indices supported?